config/locales/en.yml

```diff
@@ -30,6 +30,10 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
+  simple_form:
+    labels:
+      batch_context:
+        bundle_dir: 'Staging location'
```

Comment:
can we actually change "bundle_dir" to be "staging_location" also?

Reply:
I find "batch" "batch_context" "bundle" all confusing, TBH;  I was thinking about changing the field name, not the table name.   If you think of new hires learning this code ... having the display be diff from the var name would likely be even more confusing for them.

Reply:
There's also a "there's no better time to do this than now" element to this.

Reply:
While I agree it's worth considering, I would argue:

1. Not in this PR
2. Not without a wider discussion about names that make sense to other people

Any name change like this will touch a *lot* of code, will be a database migration, and will likely trigger other things that we will want to change.